src/fairchem/core/trainers/ocp_trainer.py

```diff
@@ -666,7 +666,9 @@ def run_relaxations(self, split="val"):
                 )
                 gather_results["chunk_idx"] = np.cumsum(
                     [gather_results["chunk_idx"][i] for i in idx]
-                )[:-1]  # np.split does not need last idx, assumes n-1:end
+                )[
+                    :-1
```

Comment:
this is really wierd linter formatting but 🤷

Reply:
That was the exact linting error... Also agree very weird, I think the IDE is injecting it sometimes